.xo-config.json

```
@@ -1,5 +1,5 @@
 {
-  "extends":["xo-react","prettier","prettier/react"],
+  "extends":["xo-react","prettier","prettier/react","plugin:jsx-a11y/recommended"],
```

Does this need the `plugins` line from the other PR?

```suggestion
  "plugins":["jsx-a11y"],
  "extends":["xo-react","prettier","prettier/react","plugin:jsx-a11y/recommended"],
```

---

Without it, it errors, but I'll just look to see what can be done.

---

To clarify, the `plugins` line that was in the other PR seems to be missing from this one. I would expect that it needs it.